### DISCUSSION TITLE
LTMEAT doesn't work on the latest chrome update

### DISCUSSION THREAD

**Discussion by obkduishinijf** [3kh0/ext-remover#882] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I charge kids $5 for me to use LTMEAT on their chromebook (scummy, i know, but atleast i'm earning something) and I just saw that LTMEAT no longer works. Javascript is still enabled, but it's not possible to run javascript on extension pages.

**Comment by S-PScripts** [comment]
javascript on bookmarklets don't work on **any** pages since some update to chromeos

**Comment by funkymokey** [comment]
charge them even more for skiovox

**Comment by DankFireR** [comment]
do the same thing but with the LTMEAT Flood method, if the computer does a soft restart tho you do need a smaller number of tabs in the folder.
edit: also how tf did it take so long for LTMeat to get patched for you??? it was patched a while ago for most people

**Reply by AshtonDavies** [reply]
Older Chromebook models probably.

**Reply by obkduishinijf** [reply]
used CAUB until my chromebook broke, got a new one and it was already updated

**Reply by funkymokey** [reply]
if only downgrading was still a viable option

**Reply by DankFireR** [reply]
ah okay, thank you @AshtonDavies
